packages/lesswrong/server/migrations/20230119T202810.user_mentions_newMentionNotifictions.ts

```diff
@@ -0,0 +1,28 @@
+import {fillDefaultValues} from '../manualMigrations/migrationUtils'
+import Users from '../../lib/collections/users/collection'
+
+/**
+ * Generated on 2023-01-19T20:28:10.210Z by `yarn makemigrations`
+ * The following schema changes were detected:
+ * -------------------------------------------
+ * ***Diff too large to display***
+ * -------------------------------------------
+ * (run `git diff --no-index schema/accepted_schema.sql schema/schema_to_accept.sql` to see this more clearly)
+ *
+ * - [ ] Write a migration to represent these changes
+ * - [ ] Rename this file to something more readable if you wish
+ * - [ ] Uncomment `acceptsSchemaHash` below
+ * - [ ] Run `yarn acceptmigrations` to update the accepted schema hash (running makemigrations again will also do this)
+ */
+export const acceptsSchemaHash = "778fab1f7c2d942253123778b67715c5";
+
+export const up = async ({db}: MigrationContext) => {
```

Comment:
@jpaddison3 or @oetherington just doublechecking this is done right.

Reply:
* These are migrations to be run on a postgres database, and will indeed not be run on any non-postgres database.
* They need to update the postgres schema, which this doesn't do. See, for example: packages/lesswrong/server/migrations/20230118T104229.add_subforumIntroPostId.ts
* You'll want to run a "manualMigrations" for a mongodb migration. That one can run the fillDefaultValues code.
* The fillDefaultValues code is still valuable here though, after the addField code.

@oetherington can correct me if I got any of that wrong.

Reply:
Yep, as JP said, you'll need to `await addField(db, Users, "notificationNewMention")`, and also `await addField(db, Comments, "pingbacks")`. Ideally, you should also put calls to `dropField` in the down migration too (the file JP mentioned has exactly the same pattern here too).

The `fillDefaultValues` is redundant here and can be safely removed. The default value is built into the column type in postgres, so it will be automatically filled when the column is created.

Reply:
I see. So as the things stand currently - I need to create both manual migrations (to update mongo) and Postgres migration in a pattern you describe, right?

And looking at the recent Slack discussion - Postgres migration would happen automatically & manualMigration would need to be run manually on prod before pushing this PR

Reply:
Yeah - unfortunately that's the case at the moment while we're switching over. It should be simpler once the process is finished.

Reply:
thanks for explanations! updated the migration code!